### PR TITLE
fix(is-visible): do not error if window.Node does not exist

### PR DESCRIPTION
**Pull request by straker**
Allows `isVisible` to be run if `window.Node` is not defined.

### CODE REVIEW

File: test/test-virtual-rules.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var path = require('path');
+var assert = require('chai').assert;
+var glob = require('glob');
+var axe = require(path.join(__dirname, '../axe'));
```
Comment (stephenmathieson) [inline test/test-virtual-rules.js:4]: Isn't this the same thing?

```suggestion
var axe = require('..');
```

Comment (straker) [reply test/test-virtual-rules.js:4]: `axe` isn't a folder so it's the `axe.js` file we're after

Comment (WilcoFiers) [inline test/test-virtual-rules.js:4]:
```suggestion
var axe = require(../axe');
```

**Review by WilcoFiers — CHANGES_REQUESTED** [review]

File: test/test-virtual-rules.js (hunk 1 above)
Comment (stephenmathieson) [reply test/test-virtual-rules.js:4]: [`axe.js` is set to `main` in `package.json`](https://github.com/dequelabs/axe-core/blob/develop/package.json#L51).  You don't need anything other than `..`. I'm 👍 for removing the `__dirname` though.

Comment (straker) [reply test/test-virtual-rules.js]: Ah. I didn't know that. I knew it looked for an `index` file when using a directory, but I didn't know a relative file path would look at `pacakge.json` as well (I only thought that was for module names).

**Review by WilcoFiers — APPROVED** [review]

**Pull request merged**